<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefineAction</class>
 <widget class="QWidget" name="DefineAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pshBtn_DefineAction">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
border: 2px solid red;
border-width : 2px;
border-color: rgb(204, 0, 0);
background-color: rgb(255, 255, 255);
color: rgb(204, 0, 0);
font: 10pt &quot;Open Sans&quot;;

}
 QPushButton:hover 
{
     background-color: rgb(204, 0, 0);
     color: rgb(255, 255, 255);
 }
QPushButton:pressed
{
	border:1px solid red;
}
QToolTip
{
	background-color: rgb(255, 255, 127);
	color:rgb(0,0,0);
}
QPushButton:disabled {
background-color:rgb(211, 215, 207);
}
</string>
       </property>
       <property name="text">
        <string>Add Action</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pshBtn_Save">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
border: 2px solid red;
border-width : 2px;
border-color: rgb(204, 0, 0);
background-color: rgb(255, 255, 255);
color: rgb(204, 0, 0);
font: 10pt &quot;Open Sans&quot;;

}
 QPushButton:hover 
{
     background-color: rgb(204, 0, 0);
     color: rgb(255, 255, 255);
 }
QPushButton:pressed
{
	border:1px solid red;
}
QToolTip
{
	background-color: rgb(255, 255, 127);
	color:rgb(0,0,0);
}
QPushButton:disabled {
background-color:rgb(211, 215, 207);
}
</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
